system/settings/settings.xml

```diff
@@ -2561,40 +2561,13 @@
         </setting>
       </group>
       <group id="2">
-        <setting id="audiooutput.dspaddonsenabled" type="boolean" label="36441" help="36438">
```

Comment:
just disable the addon. why would someone want to have an addon enable and use a different switch to disable it again?